### PR TITLE
Cameras now have a hashtable of current presets

### PR DESCRIPTION
**Pull request by tkolenbrander**

### CODE REVIEW

**Comment by stevenmeijer** [comment]
Looks good, but it needs a getter for the amount of presets

**Comment by tkolenbrander** [comment]
@TH3steven It is fixed now

**Comment by bvobart** [comment]
For consistency, the Hashtable should be a HashMap, seeing as a HashMap is used in the Script class as well.

**Comment by tkolenbrander** [comment]
@bvobart Thank you for the feedback, I will change it.

**Comment by TimYuenior** [comment]
The changes look good, but it would be useful to have an adjustable image bound to each preset. Could you add this?

**Comment by tkolenbrander** [comment]
@TimYuenior I will add a string in which you can store the location of the image

File: src/main/java/nl/tudelft/contextproject/presets/Preset.java [hunk 1]
```diff
@@ -1,5 +1,7 @@
 package main.java.nl.tudelft.contextproject.presets;
 
+import java.io.File;
```
Comment (bvobart) [inline src/main/java/nl/tudelft/contextproject/presets/Preset.java]: Unused import!

**Pull request merged**